resources/overview.js

```diff
@@ -142,7 +142,7 @@ loadJSON(json => {
         h(["div", 'class="proofinfo"'])([
           h(["div", 'class="link"'])([
             h(['a', 'class="build-hash"', `href="${o.hash}"`])(o.hash),
-            h(['span', 'class="proj-hash"'])(`proj: ${o.git}`),
+            h(['span', 'class="git-rev"'])(`rev: ${o.git.substring(0,8)}${o.branch ? ` (${o.branch})` : ''}`),
```

Comment:
What does this look like? It seems you'd get 8 chars of the hash directly attached to the branch name. Is there a space

We can just add a separate span for the branch, I think that's cleaner.

Reply:
Also I think it could be nice to trim the SHA-1 in CSS. Mostly just because it's a presentational concern.

Reply:
> Is there a space?

yes there is

>  I think it could be nice to trim the SHA-1 in CSS

How would I do that in `css`? I have no idea what I'm doing with frontend haha.

Reply:
I couldn't figure it out in 2 minutes so it's definitely not worth it, but it seems one would be able to do it with [this](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow).